### PR TITLE
Configuration tab and Interval PR

### PR DESCRIPTION
**Pull request by Sheraz064-ai**
1. Configuration tab for company communicator added.
2. Intervals functionality added for the company communicator application.
Screenshot 2021-03-10 at 11 24 36 AM

Screenshot 2021-03-11 at 5 07 47 PM

### CODE REVIEW

File: Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/NotificationData/NotificationDataRepository.cs [hunk 1]
```diff
@@ -131,8 +132,18 @@ public async Task<string> MoveDraftToSentPartitionAsync(NotificationDataEntity d
...
+                if (draftNotificationEntity.ScheduleInterval <= 1)
+                {
+                    // Delete the draft notification.
+                    await this.DeleteAsync(draftNotificationEntity);
+                }
+                else
+                {
+                    // Update schedule time and interval for next recurrence.
+                    draftNotificationEntity.ScheduleInterval -= 1;
+                    draftNotificationEntity.ScheduleDateTime += 600;
```
Comment (adeelfareed) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/NotificationData/NotificationDataRepository.cs:144]: @shoaibahmd1 This 5 mins recurring time should also be configurable, let's take it as a separate feature.

File: Source/Microsoft.Teams.Apps.CompanyCommunicator.Scheduler.Func/SchedulerFunction.cs [hunk 2]
```diff
@@ -44,11 +44,8 @@ public async Task RunAsync([TimerTrigger("0 * * * * *")]TimerInfo myTimer, ILogg
             {
                 foreach (var entity in entities)
                 {
-                    if (entity.ScheduleDateTime > 0)
```
Comment (adeelfareed) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator.Scheduler.Func/SchedulerFunction.cs:47]: @shoaibahmd1 The reason to remove this check?

**Review by adeelfareed — APPROVED** [review]
@Sheraz064-ai @shoaibahmd1 Changes look good.

File: Source/Microsoft.Teams.Apps.CompanyCommunicator.Scheduler.Func/SchedulerFunction.cs (hunk 2 above)
Comment (shoaibahmd1) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator.Scheduler.Func/SchedulerFunction.cs:47]: @adeelfareed This condition is handled in the GetEpochCondition function, so this condition seem redundant here.

File: Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/NotificationData/NotificationDataRepository.cs (hunk 1 above)
Comment (shoaibahmd1) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/NotificationData/NotificationDataRepository.cs:144]: Alright..

**Pull request merged**